docs/source/examples/Notebook/rstversions/Notebook Basics.rst

```diff
@@ -174,7 +174,7 @@ Enter command mode by pressing ``Esc`` or using the mouse to click
    </div>
 
 Mouse navigation
-================
+----------------
```

Comment:
Not changed in this PR. Though the green buttons in this section are a little confusing when running locally (unless I am missing something locally):

<img width="783" alt="screen shot 2015-12-04 at 8 30 14 am" src="https://cloud.githubusercontent.com/assets/2680980/11594891/5fc9e6e4-9a61-11e5-89db-69a92e8e0981.png">